tests/scripts/thread-cert/pktverify/packet_filter.py

```diff
@@ -347,6 +347,12 @@ def _set_found_index(self, last_index: Tuple[int, int], index: Tuple[int, int]):
         if self._parent is not None:
             self._parent._set_found_index(last_index, index)
 
+    def packet_contains_tlvs(self, p, tlvs):
+        for tlv in tlvs:
+            if tlv not in p.mle.tlv.type:
+                return False
+        return True
+
```

Comment:
```suggestion
```

I think we can't assume that TLV means MLE TLV. So the function name is not proper?
Furthermore, I think we don't need this function.